src/utils/logger.module.ts

```diff
@@ -27,7 +28,13 @@ export const configureLoggerModule = (): DynamicModule =>
           'req.headers["x-api-key"]'
         ],
         mixin: (): (() => MixinFn) => {
-          const currentTraceIds = getAPMInstance().currentTraceIds
+          let currentTraceIds = getAPMInstance().currentTraceIds
+          if (Object.keys(currentTraceIds).length === 0) {
```

Comment:
extract variable to explain the intention

Reply:
no habla inglès